roles/ci_gen_kustomize_values/templates/bgp/network-values/values.yaml.j2

```diff
@@ -0,0 +1,22 @@
+---
+# source: bgp/network-values/values.yaml.j2
+data:
+  # nodes
+{% for node_id in range(3) %}
```

Comment:
Why not base this range in the `original_content` variable? You can check the amount of nodes to patch there, allowing easy scaling if needed at some point.